Airports/EGXZ/Airspace.txt

```diff
@@ -1,77 +1,73 @@
+EGXZ Topcliffe MATZ N054.07.21.933 W001.22.09.325 N054.07.20.796 W001.22.53.860
+EGXZ Topcliffe MATZ N054.07.20.796 W001.22.53.860 N054.07.21.933 W001.23.38.396
+EGXZ Topcliffe MATZ N054.07.21.933 W001.23.38.396 N054.07.25.337 W001.24.22.594
+EGXZ Topcliffe MATZ N054.07.25.337 W001.24.22.594 N054.07.30.980 W001.25.06.121
+EGXZ Topcliffe MATZ N054.07.30.980 W001.25.06.121 N054.07.38.821 W001.25.48.648
+EGXZ Topcliffe MATZ N054.07.38.821 W001.25.48.648 N054.07.48.800 W001.26.29.852
+EGXZ Topcliffe MATZ N054.07.48.800 W001.26.29.852 N054.08.00.842 W001.27.09.420
+EGXZ Topcliffe MATZ N054.08.00.842 W001.27.09.420 N054.08.14.855 W001.27.47.548
+EGXZ Topcliffe MATZ N054.08.14.855 W001.27.47.548 N054.08.30.733 W001.28.22.468
+EGXZ Topcliffe MATZ N054.08.30.733 W001.28.22.468 N054.08.48.357 W001.28.55.392
+EGXZ Topcliffe MATZ N054.08.48.357 W001.28.55.392 N054.09.07.592 W001.29.25.577
+EGXZ Topcliffe MATZ N054.09.07.592 W001.29.25.577 N054.09.28.293 W001.29.52.791
+EGXZ Topcliffe MATZ N054.09.28.293 W001.29.52.791 N054.09.50.302 W001.30.16.829
+EGXZ Topcliffe MATZ N054.09.50.302 W001.30.16.829 N054.10.13.454 W001.30.37.505
+EGXZ Topcliffe MATZ N054.10.13.454 W001.30.37.505 N054.10.37.572 W001.30.54.662
+EGXZ Topcliffe MATZ N054.10.37.572 W001.30.54.662 N054.11.02.472 W001.31.08.167
+EGXZ Topcliffe MATZ N054.11.02.472 W001.31.08.167 N054.11.27.967 W001.31.17.916
+EGXZ Topcliffe MATZ N054.11.27.967 W001.31.17.916 N054.11.53.861 W001.31.23.833
+EGXZ Topcliffe MATZ N054.11.53.861 W001.31.23.833 N054.12.19.959 W001.31.25.871
+EGXZ Topcliffe MATZ N054.12.19.959 W001.31.25.871 N054.12.46.615 W001.31.24.125
+EGXZ Topcliffe MATZ N054.12.46.615 W001.31.24.125 N054.13.11.969 W001.31.18.269
+EGXZ Topcliffe MATZ N054.13.11.969 W001.31.18.269 N054.13.37.486 W001.31.08.682
+EGXZ Topcliffe MATZ N054.13.37.486 W001.31.08.682 N054.14.02.417 W001.30.55.325
+EGXZ Topcliffe MATZ N054.14.02.417 W001.30.55.325 N054.14.26.571 W001.30.38.295
+EGXZ Topcliffe MATZ N054.14.26.571 W001.30.38.295 N054.14.49.766 W001.30.17.722
+EGXZ Topcliffe MATZ N054.14.49.766 W001.30.17.722 N054.15.11.823 W001.29.53.760
+EGXZ Topcliffe MATZ N054.15.11.823 W001.29.53.760 N054.15.32.575 W001.29.26.592
+EGXZ Topcliffe MATZ N054.15.32.575 W001.29.26.592 N054.15.51.862 W001.28.56.423
+EGXZ Topcliffe MATZ N054.15.51.862 W001.28.56.423 N054.16.09.538 W001.28.23.483
+EGXZ Topcliffe MATZ N054.16.09.538 W001.28.23.483 N054.16.25.468 W001.27.48.235
+EGXZ Topcliffe MATZ N054.16.25.468 W001.27.48.235 N054.16.39.528 W001.27.10.313
+EGXZ Topcliffe MATZ N054.16.39.528 W001.27.10.313 N054.16.51.613 W001.26.30.641
+EGXZ Topcliffe MATZ N054.16.51.613 W001.26.30.641 N054.17.01.629 W001.25.49.310
+EGXZ Topcliffe MATZ N054.17.01.629 W001.25.49.310 N054.17.09.500 W001.25.06.637
+EGXZ Topcliffe MATZ N054.17.09.500 W001.25.06.637 N054.17.15.166 W001.24.22.947
+EGXZ Topcliffe MATZ N054.17.15.166 W001.24.22.947 N054.17.18.583 W001.23.38.575
+EGXZ Topcliffe MATZ N054.17.18.583 W001.23.38.575 N054.17.20.000 W001.22.54.000
```

Comment:
```suggestion
@ARC(region EGXZ Topcliffe MATZ centre N054.12.20.260 W001.22.53.860 radius 5)
```
Sorry, should've caught this before you'd finished the circle. If we could use the new ARC macro for circles from now on, it'll make maintaining them much easier :)